h2o-hadoop-3/assemblyjar.gradle

```diff
@@ -95,7 +95,7 @@ shadowJar {
     relocate 'org.joda.time', 'ai.h2o.org.joda.time'
     exclude hadoopShadowJarExcludes
     relocate 'com.google.common', 'ai.h2o.com.google.common'
-    relocate 'org.eclipse.jetty', 'ai.h2o.org.eclipse.jetty'
+    relocate 'org.eclipse.jetty', 'water.org.eclipse.jetty'
```

Comment:
I would steer away of any package that we internally use, what about something like `shadow`+`water` =  `darkwater`?

Reply:
I'm fine with anything. Just needed to put there something. Easily changeable. I'll wait for others as well so that I don't change it million times.

Reply:
Changed for now to `darkwater`.